### ISSUE TITLE
Issue with the new version: 9.0.0

### ISSUE THREAD

**Issue by sefinek24** [beaugunderson/ip-address#166]
Is it normal?

```
Error: Cannot find module 'C:\GitHub\npm\geoip-lite2\node_modules\ip-address\dist\ip-address.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:438:19)
    at Module._findPath (node:internal/modules/cjs/loader:680:18)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1063:27)
    at Module._load (node:internal/modules/cjs/loader:922:27)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:119:18)
    at Object.<anonymous> (C:\GitHub\npm\geoip-lite2\utils-unminified\updatedb.js:24:32)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32) {
  code: 'MODULE_NOT_FOUND',
  path: 'C:\\GitHub\\npm\\geoip-lite2\\node_modules\\ip-address\\package.json',
  requestPath: 'ip-address'
}

Node.js v18.18.0
```

**Comment by thib3113** [comment]
the project seems published without being builded ... https://www.npmjs.com/package/ip-address?activeTab=code . 

Only the source are included

**Comment by beaugunderson** [comment]
apologies, fix coming momentarily

**Comment by beaugunderson** [comment]
fixed in 9.0.5, issue was with `incremental: true` in `tsconfig.json`... `tsc` was not rebuilding the dist files during the build stage 🙃

**Comment by sefinek24** [comment]
Thanks
